po/tr.po

```diff
@@ -1818,24 +1815,23 @@ msgid "URLs"
 msgstr "Adresler"
 
 #: src/utils.cpp:783
-#, fuzzy, c-format
+#, c-format
 msgid "Failed to open file: %s"
-msgstr "Dosya açılamadı (%s)"
+msgstr "Dosya açılamadı: %s"
 
 #: src/utils.cpp:787
-#, fuzzy, c-format
+#, c-format
 msgid "Failed to read line %u: %s"
-msgstr "%u satırı okunamadı (%s)"
+msgstr "%u. satır okunamadı: %s"
```

Comment:
Just to make it clear: `%u` is replaced by a line number, so in English, it looks something like "Failed to read lilne 42: I'm tired :)". Is it correct to put the line number at the front of the message, and add a period after it?

Reply:
Latter variant is correct. This is an error in the original translation, corrected version translates as "Failed to read Xth line" (as it would be the proper way to read in Turkish). A period is used to indicate st-nd-th in Turkish.

Reply:
Wow, interesting! Got it, thanks for the explanation!